hare/algorithm.go

```diff
@@ -1,3 +1,4 @@
+// Package hare implements the Hare Protocol.
```

Comment:
can you try to run `go vet` on this package?

Reply:
Run and fixed.
It is not go vet it is "golint".